src/main/no/nav/pto/veilarbfilter/config/Configuration.kt

```diff
@@ -11,13 +11,14 @@ private const val notUsedLocally = ""
 private val defaultProperties = ConfigurationMap(
     mapOf(
         "NAIS_CLUSTER_NAME" to notUsedLocally,
-        "ISSO_JWKS_URL" to "https://isso-q.adeo.no/isso/oauth2/connect/jwk_uri",
-        "ISSO_ISSUER" to "https://isso-q.adeo.no:443/isso/oauth2",
+        "ISSO_JWKS_URL" to notUsedLocally,
+        "ISSO_ISSUER" to notUsedLocally,
         "veilarbfilter_DB_URL" to "jdbc:postgresql://localhost:54321/veilarbfilter",
         "veilarbfilter_DB_NAME" to "veilarbfilter",
         "veilarbfilter_DB_USERNAME" to "user",
         "veilarbfilter_DB_PASSWORD" to "password",
-        "VAULT_MOUNT_PATH" to notUsedLocally
+        "VAULT_MOUNT_PATH" to notUsedLocally,
+        "SECURITY_TOKEN_SERVICE_DISCOVERY_URL" to notUsedLocally
```

Comment:
Aha, was this the issue?

Reply:
nah, I returned properties to yaml files...at least for now not to waste time on investigation why envFrom is not working or maybe something else...anyway, I need to add more changes to this PR :)